image/tarexport/tarexport.go

```diff
@@ -15,10 +16,11 @@ const (
 )
 
 type manifestItem struct {
-	Config   string
-	RepoTags []string
-	Layers   []string
-	Parent   image.ID `json:",omitempty"`
+	Config       string
+	RepoTags     []string
+	Layers       []string
+	Parent       image.ID                                  `json:",omitempty"`
+	LayerSources map[layer.DiffID]*distribution.Descriptor `json:",omitempty"`
```

Comment:
I suppose you could use `digest.Digest`, since both `DiffID`s and OCI foreign sources are digests of some sort?